app/models/miq_worker_type.rb

```diff
@@ -9,12 +9,11 @@ class MiqWorkerType < ApplicationRecord
   KILL_PRIORITY_SCHEDULE_WORKERS          = 80
   KILL_PRIORITY_PRIORITY_WORKERS          = 90
   KILL_PRIORITY_WEB_SERVICE_WORKERS       = 100
-  KILL_PRIORITY_REFRESH_CORE_WORKERS      = 110
-  KILL_PRIORITY_VIM_BROKER_WORKERS        = 120
-  KILL_PRIORITY_EVENT_CATCHERS            = 130
-  KILL_PRIORITY_UI_WORKERS                = 140
-  KILL_PRIORITY_REMOTE_CONSOLE_WORKERS    = 150
-  KILL_PRIORITY_COCKPIT_WS_WORKERS        = 160
+  KILL_PRIORITY_VIM_BROKER_WORKERS        = 110
```

Comment:
@agrare @Fryguy should we remove the VIM_BROKER_WORKERS setting now that the VimBroker has been removed since Jansa?

Reply:
Oh good eye... I'll double check that